.prettierrc.yml

```diff
@@ -2,6 +2,8 @@ singleQuote: true
 trailingComma: es5
 semi: false
 proseWrap: always
+endOfLine: auto
```

Comment:
Does keeping the default value for `endOfLine` produce lots of changes in the PR? If not, I prefer to have the default value (`lf`) to not have mixed end of line flavors.

```suggestion
```

Reply:
Yes, the diff was huge, so I set it back to the old value.

Reply:
If I'm reading your comment correctly, it sounds like you're on board with saving this change for a future PR.